.github/actions/spelling/Spelling.md

```diff
@@ -0,0 +1,26 @@
+# Spell checking
```

Comment:
The file will be renamed to `advice.md` when the review/changes is otherwise complete so that it's displayed in the spelling bot report according to https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples%3A-advice.